app/models/manageiq/providers/redhat/infra_manager/ovirt_services/strategies/v3.rb

```diff
@@ -171,7 +171,7 @@ def start_clone(source, clone_options, phase_context)
 
     def vm_start(vm, cloud_init)
       vm.with_provider_object do |rhevm_vm|
-        rhevm_vm.start { |action| action.use_cloud_init(true) if cloud_init }
+        rhevm_vm.start { |action| action.use_cloud_init(true) unless cloud_init.empty? }
```

Comment:
You can probably use `if cloud_init.present?`

Reply:
OK